registration/forms.py

```diff
@@ -43,7 +43,7 @@ class BetaTestApplicationForm(NgModelFormMixin, NgFormValidationMixin, NgModelFo
     """
     class Meta:
         model = BetaTestApplication
-        exclude = ('user', 'status')
+        exclude = ('user', 'status', 'instance')
```

Comment:
@bradenmacdonald In this particular case, I had quite a few merge conflicts, since an entire directory was renamed, and I would have needed to either resolve them twice or get `git rerere` to work correctly with them, both of which seemed too much hassle.

Even when rebasing, git shouldn't drop my changes just because the file isn't there anymore; and it usually doesn't, but in this particular case it somehow did.